contracts/wrapper/NameWrapper.sol

```diff
@@ -20,6 +21,7 @@ error LabelMismatch(bytes32 labelHash, bytes32 expectedLabelhash);
 error LabelTooShort();
 error LabelTooLong(string label);
 error IncorrectTargetOwner(address owner);
+error ZeroAddress();
```

Comment:
```suggestion
error CannotUpgrade();
```

`ZeroAddress` doesn't explain the problem - the problem is that upgrades are not permitted at this time.